GeneralsMD/Code/Libraries/Source/WWVegas/WWLib/BUFF.H

```diff
@@ -46,7 +46,7 @@
 **	November of '94. Until the compiler supports this, use the following
 **	definition.
 */
-#include	"bool.h"
+//#include	"bool.h"
```

Comment:
The file is deleted. Just remove this line or is there any need to keep it?

Several times in this review.

Reply:
cleaner diff than deletions